README.md

```diff
@@ -35,35 +35,30 @@
 # After
 
 + BotD is a browser library for JavaScript bot detection
-+ Easily add ability to detect automation tools, browser spoofing and virtual machines
-+ Requires adding only 3 lines of JavaScript on your website
++ Easily add ability to detect automation tools
++ Requires adding only a few lines of JavaScript on your website
 ```
 
-### 🔩 [Try Demo](https://fingerprint.com/products/bot-detection/) - see the live demo running in your browser
+#### 🔩 [Try Demo](https://fingerprint.com/products/bot-detection/) - see the live demo running in your browser
 
-BotD runs in the [browser](#install-from-cdn); additionally you can harden it by using our open source cloud integrations.
+BotD runs in the [browser](#install); additionally you can harden it by using our open source cloud integrations.
 
-### Cloud Integrations ☁️
-
-* [CloudFlare](https://github.com/fingerprintjs/botd-integrations/tree/main/cloudflare) - runs in CloudFlare workers for increased accuracy and security.
-* [Fastly](https://github.com/fingerprintjs/botd-integrations/tree/main/fastly/wasm) - runs in Fastly Compute@Edge high-performance WASM edge
-* [Next.js/Vercel](https://github.com/vercel/examples/tree/main/edge-functions/bot-protection-botd)<sup>3rd party</sup> - runs as a Next.js edge middleware
+## 🤖 **_Check for more advanced and accurate bot detection [here](https://fingerprint.com/products/bot-detection/)_**
 
 ## Install from CDN
 
 ```html
 <script>
     // Initialize an agent at application startup.
-    const botdPromise = import('https://openfpcdn.io/botd/v0.1')
-        .then( Botd => Botd.load({ publicKey: '<your-public-key>' }))
-    // Get the bot detection result when you need it.
-    // Result will contain the `requestId` property, that you can securely verify on the server.
+    const botdPromise = import('https://openfpcdn.io/botd/v1.0').then((Botd) => Botd.load())
```

Comment:
I recommend using `v1` instead of `v1.0`. The first number is the major version, which means that the JS API is backward compatible on updates. It will ensure that the customers use the latest possible version of the library that is compatible with their code.